<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmXyzTrack</class>
 <widget class="QWidget" name="frmXyzTrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="wt_xyzPlot" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>620</width>
     <height>330</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="rb_x">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>X轴</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rb_y">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Y轴</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rb_z">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Z轴</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="sp_dnum">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_pCtrl">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>开始绘图</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_clear">
      <property name="text">
       <string>重置画布</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
